spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/base.kt

```diff
@@ -0,0 +1,97 @@
+package org.spekframework.intellij
+
+import com.intellij.execution.CommonProgramRunConfigurationParameters
+import com.intellij.execution.configuration.EnvironmentVariablesComponent
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.execution.configurations.ModuleBasedConfiguration
+import com.intellij.execution.configurations.RunConfigurationModule
+import com.intellij.openapi.util.JDOMExternalizerUtil
+import org.jdom.Element
+import org.jetbrains.kotlin.idea.KotlinIcons
+import org.spekframework.spek2.runtime.scope.Path
+import org.spekframework.spek2.runtime.scope.PathBuilder
+import org.spekframework.spek2.runtime.scope.isRoot
+
+abstract class SpekBaseConfigurationType(id: String, displayName: String): ConfigurationTypeBase(
+    id,
+    displayName,
+    "Run specifications",
```

Comment:
Do you want to move localizations to a single place? Pretty sure IJ provides a way to do this.

Reply:
Would prefer to leave it as is as of the moment, localization is the least of our problems 😂